### DISCUSSION TITLE
self-running presentation

### DISCUSSION THREAD

**Discussion by alos-source** [marp-team/marp#396] [Ideas]
I would like to create a self-running (html-)prestation with marp. There are two use-cases:
- PP already has this kind of feature for [kiosk applications](https://support.microsoft.com/en-us/office/create-a-self-running-presentation-57fc41ae-f36a-4fb5-94a3-52d5bc466037) with slides running in an endless loop.
- I would rather like to show slides in a "Pecha Kucha"-style: 20 slides, automatically switching after 20 seconds. After the last slide, the presentation just ends.

Does anyone now how to do this for html-presentations with marp?

**Comment by yhatt** [comment]
We have an open ecosystem. It's up to you to make a HTML with the presentation style you need.

Render Markdown into HTML with using [Marp Core](https://github.com/marp-team/marp-core), and write a JavaScript to navigate pages with your style. You also can use one of third-party libraries for the page navigation, such as [Bespoke.js](https://markdalgleish.com/projects/bespoke.js/) + [plugins](https://www.npmjs.com/search?q=keywords:bespoke-plugin) (just like as HTML generated from Marp CLI), [Swiper](https://swiperjs.com/), [reveal.js](https://revealjs.com/), and so on.

**Comment by mfouesneau** [comment]
Could you provide an example of how to "autoplay" slides?
